packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -51,7 +51,10 @@ export type MetricEventVerb =
   | 'abort'
   | 'sync'
   | 'login'
-  | 'logout';
+  | 'logout'
+  | 'answer'
+  | 'activate'
+  | 'deactivate';
```

Comment:
Should we try and bring in all the WxCC-specific verbs in a separate variable and export the merged list?

Reply:
I think the point is that all these verbs should be common to both the Webex web clinet and WxCC, and any other future consumers, I think we're best to stick with a single list.

Reply:
yes, we should stick to 1 list